src/views/user/ChangePassword.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div>
+    <form>
+        <b-row class="mb-3">
+            <h4>{{ $t('navbar.changePassword') }}</h4>
+        </b-row>
+        <b-row class="mb-3">
+            <b-col cols="2">{{ $t('navbar.currentPassword') }}</b-col>
+            <b-col cols="2"><b-form-input v-model="currentPassword" type="password"></b-form-input></b-col>
+        </b-row>
+        
+        <b-row class="mb-3">
+            <b-col cols="2">{{ $t('navbar.newPassword') }}</b-col>
+            <b-col cols="2"><b-form-input v-model="newPassword" type="password"></b-form-input></b-col>
+        </b-row>
+        
+        <b-row class="mb-3">
+            <b-col cols="2">{{ $t('navbar.repassword') }}</b-col>
+            <b-col cols="2"><b-form-input v-model="rePassword" type="password"></b-form-input></b-col>
```

Comment:
Maybe style this just a bit when `rePassword != ""` and `disableChange`, so the user sees pretty quickly that what has been typed in does not match newPassword.  You could maybe pt a red dot next to it, maybe make the text grey instead of black?

Reply:
We will write a small message: "Passwords must be the same"

Reply:
You got the styling set properly in the registration and the password change components, but it still needs to be added to this component.

Reply:
Should be OK now.

Reply:
Added